blog/app/controllers/articles_controller.rb

```diff
@@ -1,51 +1,50 @@
 class ArticlesController < ApplicationController
-	
-  http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
-  def index
-    @articles = Article.all
-  end
- 
-  def show
-    @article = Article.find(params[:id])
-  end
- 
-  def new
-    @article = Article.new
-  end
- 
+  
+  http_basic_authenticate_with name: "ikka", password: "ikka340", except: [:index, :show]
+
+	def new 
+		@article = Article.new
+	end
+  
   def edit
-    @article = Article.find(params[:id])
+  	@article = Article.find(params[:id])
```

Comment:
do not use .find(:id). -> This can raise a 404 exception.
Instead use find_by_id. and handle nil.
Same things on other places also